packages/widgets/src/lib/components/InferenceWidget/shared/WidgetTextarea/WidgetTextarea.svelte

```diff
@@ -29,10 +29,14 @@
 		}
 		await tick();
 		// split on whitespace or any other character to correctly render newlines \n
-		for (const char of outputTxt.split(/(\s|.)/g)) {
-			await delay(typingEffectSpeedMs);
-			spanEl.textContent += char;
-			moveCaretToEnd();
+		if (typingEffect) {
+			for (const char of outputTxt.split(/(\s|.)/g)) {
+				await delay(typingEffectSpeedMs);
+				spanEl.textContent += char;
+				moveCaretToEnd();
```

Comment:
While you're at it, can you please remove the `containerSpanEl?.focus();` at L44 please ? 🙏 

It's super annoying because it prevents any interaction with the Hub while the widget is displaying generated code

Reply:
handled in a65eb93c65c089f2d8d29c0a687b05a421d38f22

Reply:
handled here https://github.com/huggingface/huggingface.js/pull/397/commits/24317f7da8352b433b728412676d75c1712a3c9d